.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+language: python
+python:
+  - '2.6'
+  - '2.7'
+  - '3.4'
```

Comment:
At first I just wanted to get the current CI setup working, I plan to open a new PR adding new envs/dropping old ones if we decide to (see my comment on the PR itself).